### DISCUSSION TITLE
How can I retrieve the xterm.js version string with a js api?

### DISCUSSION THREAD

**Discussion by ceccopierangiolieugenio** [sourcelair/xterm.js#5195] [Q&A] [answered]
I am working on a project built around xterm.js and pyodide:
https://ceccopierangiolieugenio.github.io/pyTermTk/sandbox/sandbox.html

I manage to print the pyodide version using:
```js
this.pyodide.version
```

is it a similar field/api I can use in xterm.js to retieve the version number?

Thanks

**Comment by jerch** [comment, marked as answer]
Nope, the bundled terminal code does not carry this information as far as I am aware. As integrator you'd normally get the version from the package.json.
This is indeed problematic for cmdline apps, thats why I created #4982. We also had similar older ideas, which none of them made it into existance.

**Comment by ceccopierangiolieugenio** [comment]
Thanks, so far it is not a big issue in my project,
I hope eventually this will be included.
